config/initializers/rack_attack.rb

```diff
@@ -0,0 +1,13 @@
+LIMIT_LOGIN_ATTEMPTS_PER_IP = 16
```

Comment:
Is it worth having this as an env var? or maybe we should wait until we need variations per environments

Reply:
I think it's fine like this until we need per-environment values